assignments/callbacks.js

```diff
@@ -27,23 +27,28 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.pop());
```

Comment:
`.pop()` actually modifies the array by removing the last element it returns. Is this desired behavior?